draft-ietf-teep-protocol.md

```diff
@@ -320,7 +320,7 @@ the following features are supported:
  - Request for attestation information,
  - Listing supported extensions,
  - Querying installed Trusted Components, and
- - Listing supported SUIT commands.
+ - Request for logging information in SUIT Reports.
 
```

Comment:
In this PR, passing the attestation of TAM to Agent is a new feature.
This QueryRequest feature list should have the new one as like 
```
- Providing the attestation data of TAM to TEEP Agent
```

Reply:
@ko-isobe 
Thanks for your comment. 

I added that it will be sending attestation data of the TAM to the TEEP Agent in a separate sentence, since the data-item-requested section is for what to request from the TEEP Agent and not what the TAM would send.